### ISSUE TITLE
documentation

### ISSUE THREAD

**Issue by rmgk** [rmgk/viscel#12]
it would be nice to have
- [ ] what is viscel good for
- [ ] how to start
  - sbt run
- [ ] how to find the interface
- [ ] what the heck is happening
- [ ] how to fix stuff that breaks. which it will.
- [ ] how to write new narrators (that probably depends on #10 as no one is going to understand the full versions the first time :)
- [ ] overview over the different parts

**Comment by rmgk** [comment]
readme has some parts … lets say thats good enough for now 😅
